cxplat/cxplat_test/cxplat_fault_injection.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
Right now the file naming convention is that cxplat_*_test.cpp tests cxplat_* functionality.
So I think this file should end in *_test.cpp